testy.js

```diff
@@ -65,11 +67,16 @@ class Testy {
   }
   
   _loadAllRequestedFiles() {
-    this._resolvedTestFilesPathsToRun().forEach(path =>
-      Utils.allFilesMatching(path, this._testFilesFilter()).forEach(file =>
-        require(file)
-      )
-    );
+    try{
+      this._resolvedTestFilesPathsToRun().forEach(path =>
+        Utils.allFilesMatching(path, this._testFilesFilter()).forEach(file =>
+          require(file)
+        )
+      );
+    }catch(err){
```

Comment:
I think we need to catch the specific "file not found" error, not any type of error